alerts/cluster-monitoring-operator/KubeDaemonSetMisScheduled.md

```diff
@@ -0,0 +1,104 @@
+# KubeDaemonSetMisScheduled
+
+## Meaning
+
+The `KubeDaemonSetMisScheduled` alert triggers when, over a defined time interval,
+an excessive number of daemonset pods are found running on nodes where they
+should not be running. This condition typically occurs when a pod is scheduled
+on a node with a taint that doesn't have a corresponding toleration, or when a
+node is unhealthy or overloaded, which can hinder pod scheduling.
+
+## Impact
+
+Daemonset pods are running where they are not supposed to run. Excessive resource
+usage where they could be used by other apps.
+
+## Diagnosis
+
+- Check Daemonset status
+
+  ```console
+  oc -n $NAMESPACE describe daemonset $NAME
+  ```
+
+  Replace $NAME with the name of the Daemonset having the issue.
+  $NAMESPACE comes along with alert labels, replace it with respective value
+  from the alert received.
+
+- Check the status of the pods belonging to the Daemonset and look for the `Node`,
+  `Tolerations`, and `NodeSelector` fields:
+
+  ```console
+  oc -n $NAMESPACE describe pod $NAME
+  ```
+
+  Replace $NAME with the name of the pod having the issue.
+
+  - Look for the `Node`, `Node-Selectors` and `Tolerations` fields in the output.
+
+  - Check the labels for the node and verify with `NodeSelector` on the pod.
```

Comment:
```suggestion
  - Check the labels for the node and verify that they match the `NodeSelector` values for the pod.
```
Not sure if this is what you mean, but it's unclear to me what "verify with" means here.